Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/PrefabProcessorContext.cpp

```diff
@@ -30,7 +29,14 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
 
     PrefabProcessorContext::PrefabProcessorContext(const AZ::Uuid& sourceUuid)
         : m_sourceUuid(sourceUuid)
-    {}
+    {
+        AZ::Interface<EntityIdPathMapperInterface>::Register(this);
```

Comment:
Tip: you can achieve the same thing by inheriting from AZ::Interface::Registrar.